tests/test_pagination.py

```diff
@@ -44,6 +44,16 @@ def testable_streams_static(self):
             'locations',  # This stream does not paginate in the sync (See Above)
         })
 
+    def cleanup_excess(self):
+        cleanup = {'categories': 10000}
+        for stream, limit in cleanup.items():
+            print("Checking if cleanup is required.")
+            all_records = self.client.get_all(stream, start_date=self.STATIC_START_DATE)
```

Comment:
Any reason to use the static start date here and not None or something like that? Does it work if you don't pass a start_time here? I suspect it does but not sure.

Reply:
I assumed a start date was required. Checking now

Reply:
The test method for get_all does not work without a start_date param. And I don't think the tap should. Actually, not sure if it matters one way or the other.

Reply:
Ok it's fine, just leave it.